### PR TITLE
Development

### PR DESCRIPTION
**Pull request by kobiyoyo**
Code Review

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Great work team 👍
But I notice that you cloned the whole page instead of doing the grayscale heatmap. I hope that by doing this you learned a lot about visual hierarchy, typography, and design principles. Before merging this project to master, we need to do some changes to be in line with Microverse guidelines.

- [ ] You folks used the main and section tag in the right way. But you also need to use the article tag for each one of the articles in this page.
- [x] All content should be wrapped with `main` HTML tags
- [x] Every child of main should be a `section` HTML tag pair
- [ ] Every child of section should be an `article` HTML tag pair
- [ ]  There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

- [ ]  There are some errors in your CSS code. Check them using this CSS [validator](https://jigsaw.w3.org/css-validator/) . Ignore the warnings.

Please make these changes and submit for code review again.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+
+                                            </div>
+                                         </div>
+                                       </a>
+                            </article>
+
+                        </section> 
+                        <section class="thirdContent">
+                        <p>Our friends and supporters ↓</p>
+                        <div class="blackAdWrap">
```
Comment (maelfosso) [inline index.html]: Please use `<article>` here. Also, Put the `<p>Our friends... </p>` into.

**Comment by maelfosso** [comment]
Hello guys,

- [ ] Please correct errors from https://validator.w3.org

- [ ]  There are some errors in your CSS code. Check them using this CSS validator https://jigsaw.w3.org/css-validator/. Ignore the warnings

- [ ] Correct errors mentioned by `stickler-ci`

- [x] I also comment some part to change

- [x] Please, put the `<footer>` outside the `<main>`. Do the samething for the `<nav>`.

Kindly make those changes and submit a new code review request.
Thanks.

**Comment by HenryTabima** [comment]
# Good Job guys!

You achieved the most important things so this is a thumb up :+1: , submit the respective project completion form.

## Observations: 
* The stickler is still complaining about some things
* Is better to have the font request in the HTML. you can even make that request asynchronously

:computer: Happy Coding

**Pull request merged**